apps/web/.gitignore

```diff
@@ -2,3 +2,4 @@
 build
 public/build
 .env
+*.js
```

Comment:
Is it needed?

Reply:
Needed to ignore the built`server.js`. Should we not?